contacts_theme.theme

```diff
@@ -70,8 +70,49 @@ function contacts_theme_preprocess_block(&$variables) {
     ];
     foreach ($variables['content'] as $key => &$content) {
       if (in_array($key, $keys)) {
-        $content['button']['#value'] = '';
+        $content['#attributes']['class'] = array('dropdown');
+        $content['#prefix'] = '<div class="action-group-wrapper dropdown">';
+        $content['#suffix'] = '</div>';
+        $content['button']['#value'] = '+';
```

Comment:
So I think this and the `button-action-icon` should only be applied to the create group